src/pages/index.html

```diff
@@ -29,14 +32,25 @@
                 <source src="../public/video/head_vid.mp4" type="video/mp4">
                 Your browser does not support the video tag.
             </video>
+            
+            <!-- hero text-->
+            <div class ="hero-text">
+                <h2>The World's Best Preschool Adventure Park!</h2>
+            </div>
 
             <!-- Open hours & location -->
             <aside class="aside-hour-location-container">
                 <!-- Using divs to apply flex justify-content equally -->
                 <div>
                     <img src="../public/images/clock.png" class="aside-clock-icon">
-                    <span>Opening hours</span>
+                    <span>Opening hours: Monday-Saturday 8am-6pm.</span>

```

Comment:
You should add padding to this if you want more space between elements in a flex container. As `span`  might not be the right element here to apply proper padding, it's totally fine to change this to a `div`  to get the proper behaviour.